### PR TITLE
Fix exonum-java scripts and clarify their usage [ECR-3094]

### PR DESCRIPTION
**Pull request by vitvakatu**
## Overview

---
See: https://jira.bf.local/browse/ECR-3094

### Definition of Done

- [ ] There are no TODOs left in the code
- [ ] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [ ] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [ ] Public API has Javadoc
- [ ] Method preconditions are checked and documented in the Javadoc of the method
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

### CODE REVIEW

File: exonum-java-binding/core/rust/exonum-java/start_cryptocurrency_node.sh [hunk 1]
```diff
@@ -49,29 +46,29 @@ header "PREPARE PATHS"
 
 EJB_LOG_CONFIG_PATH="${EJB_APP_DIR}/log4j-fallback.xml"
 
-export LD_LIBRARY_PATH="$JVM_LIB_PATH"
+export LD_LIBRARY_PATH="$JAVA_LIB_DIR"
```
Comment (dmitry-timofeev) [inline exonum-java-binding/core/rust/exonum-java/start_cryptocurrency_node.sh]: I wonder if it is OK to use `JAVA_LIB_DIR` variable from the tests_profile :thinking: I considered setting RUSTFLAGS to be the only effect of tests_profile, not these intermediate variables. Shall we make it required to set this variable as well?

Comment (vitvakatu) [reply exonum-java-binding/core/rust/exonum-java/start_cryptocurrency_node.sh]: These scripts are effectively protected from unset variables by `set -eu -o pipefail`. I think we can safely rely on the current tests_profile implementation, as these scripts are intended to be used by us (and any problem will be spotted just in time).

File: exonum-java-binding/core/rust/exonum-java/start_cryptocurrency_node.sh [hunk 2]
```diff
@@ -1,4 +1,10 @@
 #!/usr/bin/env bash
+# Runs exonum-java executable from target directory and starts one Exonum node with
+# cryptocurrency service.
+#
+# This script is intended to be used by Exonum developers to speed up working workflow.
```
Comment (skletsun) [inline exonum-java-binding/core/rust/exonum-java/start_cryptocurrency_node.sh]: "working workflow" -> just "workflow"?

File: exonum-java-binding/core/rust/exonum-java/start_qa_node.sh [hunk 3]
```diff
@@ -1,4 +1,10 @@
 #!/usr/bin/env bash
+# Runs exonum-java executable from target directory and starts N Exonum nodes with
+# QA service, where N corresponds to the first argument of the script.
+#
+# This script is intended to be used by Exonum developers to speed up working workflow.
```
Comment (skletsun) [inline exonum-java-binding/core/rust/exonum-java/start_qa_node.sh]: Same again.

**Pull request merged**